### PR TITLE
AST-1916 - fix: Content background color does not apply to Author Box in customizer preview.

### PR DESCRIPTION
**Pull request by yssambare12**
**Description**

The content background color settings are not applied to the Author Info Box. In the customizer, it looks like the settings are applied but reverted when refresh or return to the frontend. 
I tried on a clean install.

***How to reproduce:***
1. Activate the Blog Pro module
2. Enable Author Info in Blog > Single Post
3. Set container layout as Boxed or Content Boxed
4. Apply content background color and site background (or just content background)
5. The changes are reflected fine in the customizer. 
6. Refresh or return to the frontend and the changes are gone. 

***No. of tickets:***
1
https://secure.helpscout.net/conversation/1764312679/144544/

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@ v4.0.2
 - Fix: Importing wp-starter-content template on fresh setup overriding the existing site title.
 - Fix: Dynamic customizer - Single-archive section titles and taxonomy meta names getting by its slug name instead of title.
 - Fix: Admin dashboard's Site Identity quick setting link redirecting to Header Builder > Logo section instead of redirecting to Site Identity section.
+- Fix: Content Background Color Does Not Apply To Author Info Box.
```
Comment (imnavanath) [inline changelog.txt:19]: Changelog you placed at wrong place.

Comment (imnavanath) [inline changelog.txt:19]:
```suggestion
- Fix: Content background color does not apply to Author Info Box in customizer preview.
```

Comment (yssambare12) [reply changelog.txt]: Updated.

Comment (yssambare12) [reply changelog.txt]: Updated.

**Review by neerajmasai — APPROVED** [review]

**Review by er-roshan — APPROVED** [review]

**Pull request merged**